src/deep-resource/lib/Resource/Request.js

```diff
@@ -386,9 +384,10 @@ export class Request {
    * @param {String} url
    * @param {String} httpMethod
    * @param {Object} payload
+   * @param {Function} callback
    * @private
    */
-  _createAws4SignedRequest(url, httpMethod, payload) {
+  _createAws4SignedRequest(url, httpMethod, payload, callback) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/MitocGroup/deep-framework/file/1363148735/issues/source?bid=2895475&fileBranchId=2973909#l390)